Command/DumpCommand.php

```diff
@@ -0,0 +1,250 @@
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+/**
+ * Handles the importing of the images from the command line
+ */
+class DumpCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure ()
+    {
+        $this
+            ->setName('becklyn:db:dump')
+            ->setDescription('Dumps the configured or given Databases using \'mysqldump\' to .sql files.')
+            ->addOption(
+                'databases',
+                'd',
+                InputOption::VALUE_REQUIRED,
+                'If provided it overrides the values from the <info>config.yml</info>. Multiple Database names are separated by <info>comma (,)</info>',
+                null
+            )
+            ->addOption(
+                'path',
+                'p',
+                InputOption::VALUE_REQUIRED,
+                'The folder path where the .sql file will be saved. Defaults to %kernel.root%/var/db_backups/ if not specified otherwise.',
+                null
+            )
+            ->addOption(
+                'yes',
+                null,
+                InputOption::VALUE_NONE,
+                'Suppresses prompt asking whether to continue.'
+            );
+    }
+
+
+    /**
+     * {@inheritdoc}
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     */
+    protected function execute (InputInterface $input, OutputInterface $output)
+    {
+        $selectedDatabases = $input->getOption('databases');
+        $backupPath        = $input->getOption('path');
+        $suppressPrompt    = $input->getOption('yes');
+
+        $databases = null;
+
+
+        // When the user has specified any databases (comma-separated) as parameter
+        // we use those instead of the ones that were configured in the config.yml
+        if ($selectedDatabases !== null)
+        {
+            $databases = $this->getConnectionDataByDatabaseNames(explode(',', $selectedDatabases));
+        }
+        else
+        {
+            // Use the default, configured databases from the config.yml:
+            // 1) Use 'becklyn_mysqldump:databases' when set
+            // 2) else fallback to the databases from 'doctrine:dbal:connections'
+            $databases = $this->getDatabases();
+        }
+
+        if ($backupPath === null)
+        {
+            $backupPath = $this->getBackupDirectory();
+        }
+
+        // If there are no database connection information available we can't dump anything
+        if (count($databases) === 0)
+        {
+            $output->writeln('<error>Exiting</error>: No connection data found.');
+            return;
+        }
+
+        $output->writeln('<info>»</info> Found connection data for the following databases:');
+        $output->writeln('Database(s): <info>' . implode(',', array_keys($databases)) . '</info>');
+        $output->writeln("Backup directory: <info>$backupPath</info>");
```

Comment:
Would it be better to draw a table?

Database | Backupfile
--- | --- |
example | app/var/example.sql
example2 | app/var/example2.sql
example3 | app/var/example3.sql

etc...